tsconfig.json

```diff
@@ -17,30 +17,69 @@
     "jsx": "preserve",
     "baseUrl": ".",
     "paths": {
-      "config": ["config/index"],
-      "ducks": ["src/ducks/index"],
-      "ducks/*": ["src/ducks/*"],
-      "modules": ["src/components/modules/index"],
-      "modules/*": ["src/components/modules/*"],
-      "common": ["src/components/common/index"],
-      "common/*": ["src/components/common/*"],
-      "hooks": ["src/hooks/index"],
-      "hooks/*": ["src/hooks/*"],
-      "images/*": ["src/static/images/*"],
-      "router": ["server/router"],
-      "server/*": ["server/*"],
-      "styles": ["src/styles/index"],
-      "styles/*": ["src/styles/*"],
-      "services": ["src/services"],
-      "services/*": ["src/services/*"],
-      "store": ["src/store"],
-      "types": ["src/types/index"],
-      "vectors/*": ["src/static/vectors/*"]
+      "config": [
+        "config/index"
+      ],
+      "ducks": [
+        "src/ducks/index"
+      ],
+      "ducks/*": [
+        "src/ducks/*"
+      ],
+      "modules": [
+        "src/components/modules/index"
+      ],
+      "modules/*": [
+        "src/components/modules/*"
+      ],
+      "common": [
+        "src/components/common/index"
+      ],
+      "common/*": [
+        "src/components/common/*"
+      ],
+      "hooks": [
+        "src/hooks/index"
+      ],
+      "hooks/*": [
+        "src/hooks/*"
+      ],
+      "images/*": [
+        "src/static/images/*"
+      ],
+      "router": [
+        "server/router"
+      ],
+      "server/*": [
+        "server/*"
+      ],
+      "styles": [
+        "src/styles/index"
+      ],
+      "styles/*": [
+        "src/styles/*"
+      ],
+      "services": [
+        "src/services"
+      ],
+      "services/*": [
+        "src/services/*"
+      ],
+      "store": [
+        "src/store"
+      ],
+      "types": [
+        "src/types/index"
+      ],
+      "vectors/*": [
+        "src/static/vectors/*"
+      ]
```

Comment:
Maybe nice to put all of these on one line? I know eslint copy paste rules do mess this up tho

```json
"types": ["src/types/index"
"vectors/*": ["src/static/vectors/*"]
```

Reply:
Fixed. Next messed up the file when I first launched the server with v12.